e2e/app/page/notebook-page.ts

```diff
@@ -500,7 +527,7 @@ export default class NotebookPage extends NotebookFrame {
     const fileSize = await getPropValue(fileSizeElement, 'textContent');
 
     // In case page has to be checked after finish.
-    await takeScreenshot(newPage, `notebook-upload-file-${fileName}.png`);
```

Comment:
why remove the .png ?

Reply:
I'm updating `takeScreenshot` func to create new file with default `.png` suffix. It's not needed here anymore.

Reply:
every other use of `takeScreenshot()` keeps the `.png` and `.jpg`, so we should either (a) keep going and remove the extensions from all calls or (b) revert the changes in this PR

Reply:
okay. reverted `save-file-utils.ts`